addons/web/static/tests/core/notebook.test.js

```diff
@@ -0,0 +1,357 @@
+/** @odoo-module */
+
+import { expect, test } from "@odoo/hoot";
+import { animationFrame } from "@odoo/hoot-mock";
+import { click } from "@odoo/hoot-dom";
+import { Component, xml } from "@odoo/owl";
+import { mountWithCleanup } from "@web/../tests/web_test_helpers";
+import { Notebook } from "@web/core/notebook/notebook";
```

Comment:
add a separation between imports from hoot/owl/helpers and others (notebook here).
There is a difference of loading between these modules and we want to separate them to make it clear.